modules/twitch_streaming

```diff
@@ -0,0 +1,55 @@
+
+"""
+Checks if a streamer is online. [Gross Gore or grossie_gore as example placeholder]
+
+Checks if a streamer is online using the requests library to check to see if the dictionary key 'stream' in the json response is set to None or not.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds
+    stream_name: name of streamer(twitch.tv/<stream_name>)
+    
+Format of status string placeholders
+    {stream_name}:  name of the streamer (twitch.tv/<stream_name>)
+     
+@author Alex Caswell horatioesf@virginmedia.com
+@license BSD
+"""
+
+# import your useful libs here
+from time import time
+import requests
+
+
+class Py3status:
+    # available configuration parameters
+    # can be customized in i3status.conf
+    cache_timeout = 10
+    stream_name = "grossie_gore"
+    format = "{stream_name} is live!"
+    format_offline = "{stream_name} is offline."
+
+    def is_streaming(self, i3s_output_list, i3s_config):
+        r = requests.get('https://api.twitch.tv/kraken/streams/' + self.stream_name)
+        if r.json()['stream'] == None: 
```

Comment:
if the request fails (offline etc) will this fail? 

`r.json().get('stream')` is safer

 Can `r.json()` ever be `None`?